lib/t/news.ex

```diff
@@ -136,108 +136,331 @@ defmodule T.News do
-                "position" => [24.0, 148.0],
-                "background_fill" => "#F97EB9"
-              },
-              %{
-                "value" =>
-                  dgettext(
-                    "news",
-                    "Теперь в ленте показывается\nпримерное расстояние\nдо пользователя."
-                  ),
-                "position" => [24.0, 238.0],
-                "background_fill" => "#F97EB9"
-              },
-              %{
-                "value" =>
-                  dgettext(
-                    "news",
-                    "Ты можешь включить\nавтоматическое определение\nлокации, чтобы она\nоставалась актуальной 👇"
-                  ),
-                "position" => [24.0, 356.0],
-                "background_fill" => "#F97EB9"
-              },
-              %{
-                "action" => "enable_auto_location",
-                "value" => dgettext("news", "Включить авто-локацию"),
-                "position" => [75.0, 502.0],
-                "background_fill" => "#F97EB9"
-              }
-            ],
-            "size" => [375, 667]
-          }
-        ]
-      },
-      %{
-        id: 3,
-        timestamp: ~U[2022-03-31 10:00:00Z],
-        version: "6.1.1",
-        story: [
-          %{
-            "background" => %{"color" => "#111010"},
-            "labels" => [
-              %{
-                "value" => dgettext("news", "Привет! 👋"),
-                "position" => [24.0, 80.0],
-                "background_fill" => "#F97EB9"
-              },
-              %{
-                "value" => dgettext("news", "Мы добавили\nновые контакты:"),
-                "position" => [24.0, 148.0],
-                "background_fill" => "#F97EB9"
-              },
-              %{
-                "position" => [24.0, 238.0],
-                "answer" => "getsinceapp",
-                "question" => "messenger"
-              },
-              %{
-                "position" => [24.0, 306.0],
-                "answer" => "kindly@getsince.app",
-                "question" => "imessage"
-              },
-              %{
-                "position" => [24.0, 374.0],
-                "answer" => "since_app",
-                "question" => "twitter"
-              },
-              %{
-                "position" => [24.0, 442.0],
-                "answer" => "getsince",
-                "question" => "snapchat"
-              },
-              %{
-                "position" => [24.0, 510.0],
-                "answer" => "+541176148981",
-                "question" => "signal"
-              },
-              %{
-                "value" => dgettext("news", "Общайтесь там, где удобно ✌️"),
-                "position" => [24.0, 578.0],
-                "background_fill" => "#F97EB9"
-              }
-            ],
-            "size" => [375, 667]
-          }
-        ]
       }
     ]
+
+    other_news =
+      case Gettext.get_locale() do
```

Comment:
This news was fully* done in our stories editor, so it was almost as simple as a copy-paste from db. Going forward I plan to keep producing news the same way.
Plus sometimes simple gettext("news", ...) for localization of labels value is not enough (you have to localize stuff like position due to difference in text sizes as well).

* except for "edit_story" action